pkg/proxy/conntrack/cleanup_test.go

```diff
@@ -229,24 +255,19 @@ func TestCleanStaleEntries(t *testing.T) {
 	// entriesBeforeCleanup - entriesAfterCleanup = entries cleared by conntrack reconciler
 	var entriesAfterCleanup []*netlink.ConntrackFlow
 
+	// we create 6 fake flow entries with `testOldEndpointPort`, this simulates the case when
+	// endpoints change port without changing IP. These entries should be cleared by reconciler.
+	for _, origDest := range []string{testClusterIP, testLoadBalancerIP, testExternalIP} {
```

Comment:
Does it make sense to move `[]string{testClusterIP, testLoadBalancerIP, testExternalIP}` to a const?
It's used in a few placed.